packages/evergreen-colors/src/colors.json

```diff
@@ -335,7 +335,7 @@
     "900": "#f0822d",
     "1000": "#ec601e"
   },
-  "turquoise": {
+  "teal": {
```

Comment:
following spec,  I changed this from `turquoise` to `teal`.  Can change back if you'd like.

Reply:
Oops, my fault. Let's keep `turquoise` for now. Although I like teal better, I need to make some more changes outside of this repo to make this work.